spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/util/Importer.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2009 - 2022 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.spec.bootstrap.data.util;
+
+import org.candlepin.dto.api.client.v1.AsyncJobStatusDTO;
+import org.candlepin.dto.api.client.v1.OwnerDTO;
+import org.candlepin.invoker.client.ApiException;
+import org.candlepin.spec.bootstrap.client.ApiClient;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+public final class Importer {
+
+    private final ApiClient client;
+    private final boolean async;
```

Comment:
Apologies for finding this late, but we don't need conditionals for running the test sync or async, or any of the abstractions around this, because we plan to remove the sync endpoint in the next sprint or so (it is deprecated and no longer used), so all importing will be async from now on

Reply:
Done.